modules/mbedtls/Kconfig

```diff
@@ -202,6 +202,14 @@ config MBEDTLS_HEAP_SIZE
 	  be needed. For some dedicated and specific usage of mbedtls API, the
 	  1000 bytes might be ok.
 
+config MBEDTLS_HEAP_CUSTOM_SECTION
+	bool "Use a custom section for the mbed TLS heap"
+	depends on MBEDTLS_ENABLE_HEAP
```

Comment:
I think instead of having those `depends on`, it would make more sense to have an `if`.